recipes/web.py/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "web.py" %}
+{% set version = "0.38" %}
+{% set sha256 = "09c61b5b6b6e1dd558818d07dfbab3b2da0623603079c7d7223644b287f08870" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --record record.txt
```

Comment:
Thanks, but I am now getting a new problem with 0.38:
https://travis-ci.org/conda-forge/staged-recipes/builds/259719536#L394

Therefore, I am going to use version 0.40.dev0, as explained here:
https://github.com/webpy/webpy/issues/396

So probably will have to include `--single-version-externally-managed` back again...